vmdk_plugin/utils/fs/constants.go

```diff
@@ -0,0 +1,23 @@
+// Copyright 2017 VMware, Inc. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// Platform independent fs module constants.
+
+package fs
+
+const (
```

Comment:
nit - I am not sure this is needed at all. Using "\r" instead of `cr` does not seem to be a  problem to me. At any rate, it's ok as  is too